cmd/funnel/main.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"net/http"
+	"net/http/httputil"
+	"net/url"
+	"os"
+
+	"github.com/eagraf/habitat-new/internal/node/config"
+	"github.com/eagraf/habitat-new/internal/node/logging"
+	"tailscale.com/tsnet"
+)
+
+func main() {
+	// read command line args
+	port := os.Args[1]
+	hostName := os.Args[2]
```

Comment:
nit: probably should panic first is len(os.Args) < 3

Reply:
it'll panic anyway just won't have any err msg. If we end up adding any more code to this file I'll make it more user friendly